scripts/ci-entrypoint.sh

```diff
@@ -115,11 +122,6 @@ select_cluster_template() {
         export CLUSTER_TEMPLATE="test/ci/cluster-template-prow.yaml"
     fi
 
-    if [[ -n "${TEST_CCM:-}" ]]; then
-        # replace 'prow' with 'prow-external-cloud-provider' in the template name if testing out-of-tree
-        export CLUSTER_TEMPLATE="${CLUSTER_TEMPLATE/prow/prow-external-cloud-provider}"
```

Comment:
Since the templates above (e.g., `test/dev/cluster-template-custom-builds.yaml`) now are out-of-tree how do upstream k/k tests that use the in-tree cloud-provider work w/ ci-entrypoint going forward? Is there a test-infra change queued up to work, or am I not understanding how these changes are back-compat for those test scenarios?

Reply:
`test/dev/cluster-template-custom-builds.yaml` is actually still in tree and will remain so until the cloud-provider-azure code is removed from k/k as we still need to test in tree on k/k PRs for the time being 

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/3e4da2823246799bb77ee2be0d1e12bc4f6b1cfc/templates/test/dev/custom-builds/kustomization.yaml

cc @feiskyer

Reply:
Got it, thanks

Reply:
for prow-ci-version, the changes needed to make them work with CI CCM in test-infra are already merged (and tested in this PR):
- https://github.com/kubernetes/test-infra/pull/28821
- https://github.com/kubernetes/test-infra/pull/28825